src/plugins/auto/tests/functional/shared_tests_instances/behavior/infer_request/memory_states.cpp

```diff
@@ -1,29 +0,0 @@
-// Copyright (C) 2018-2023 Intel Corporation
-// SPDX-License-Identifier: Apache-2.0
-//
-
-#include "behavior/infer_request/memory_states.hpp"
-
-#include <common_test_utils/test_constants.hpp>
-
-#include "functional_test_utils/plugin_cache.hpp"
-#include "ov_models/builders.hpp"
-
-using namespace BehaviorTestsDefinitions;
-
-namespace {
-std::vector<memoryStateParams> memoryStateTestCases = {
```

Comment:
the test isn't presented in API2.0 version. Please, prepare test in separate PR based on changes from https://github.com/openvinotoolkit/openvino/pull/21627

Reply:
JIRA created to track: https://jira.devtools.intel.com/browse/CVS-130014